package.json

```diff
@@ -6,6 +6,20 @@
     "express": "^4.13.1",
     "jade": "~1.11.0",
     "mongoose": "^4.4.6",
-    "body-parser": "^1.15.0"
+
+    "chai": "~3.5.0",
+    "connect-flash": "^0.1.1",
+    "cookie-parser": "^1.3.3",
+    "debug": "^2.1.1",
+    "express-session": "^1.10.1",
+    "morgan": "^1.5.1",
+    "mocha": "^2.4.5",
```

Comment:
We are thinking of doing manual authentication as well as google auth because admins will need manual authentication.